instance/models/openedx_appserver.py

```diff
@@ -144,6 +154,8 @@ def set_field_defaults(self):
         """
         Set default values.
         """
+        if not self.github_admin_organizations and settings.DEFAULT_ADMIN_ORGANIZATION:
```

Comment:
@haikuginger That change makes sense, since the only way to get in a situation that an admin user is set, but the list of admin orgs is empty while a default org is configured is by manually deleting the org from the list of admin orgs, in which case we don't want it to reappear.